Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_PBMC.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Combine runs run parquet files into one
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+import gc
+
+from pycytominer.cyto_utils import output
+
+
+# ## Set paths
+
+# In[2]:
+
+
+# set path to parquet directory with annotated runs
+annotated_dir = pathlib.Path("./data/annotated_data")
+
+# directory where the combined parquet file is saved to
+output_dir = pathlib.Path("./data/combined_data")
+output_dir.mkdir(exist_ok=True)
+
+# set path for combined run parquet file
+merged_runs_path = pathlib.Path(f"{output_dir}/PBMC_sc.parquet")
+
+
+# In[ ]:
```

Comment:
Looks like this cell wasn't ran, recommend rerunning to have all cells in order.